src/rpc/blockchain.cpp

```diff
@@ -2149,15 +2149,16 @@ static RPCHelpMan scantxoutset()
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
 {
     UniValue result(UniValue::VOBJ);
-    if (request.params[0].get_str() == "status") {
+    const auto action{self.Arg<std::string>("action")};
+    if (action == "status") {
```

Comment:
nit: What is the rationale for using the new Arg() overload in some locations but not others?   Increasing consistency of usage consistency may increase readability.
For example, `action == <string literal>` replaces `request.params[0].get_str()` in:
https://github.com/bitcoin/bitcoin/blob/391873694137f241dfe1f57ed9058a438e147218/src/rpc/blockchain.cpp#L2153
but not later in:

https://github.com/bitcoin/bitcoin/blob/391873694137f241dfe1f57ed9058a438e147218/src/rpc/blockchain.cpp#L2239

Reply:
Throughout the files modified for the PR, looks like there might be other opportunities to use the new overloading (e.g. other instances of `params[0].get_str()`).  Some examples are referenced below.  It would seem reasonable to scope this PR for the overloading and initial usage, and batch more updates in a subsequent PR (helps keep this PR small/digestible).  It's your call as the PR author.  Great PR, and I would support either approach.

In `scanblocks()`:
https://github.com/bitcoin/bitcoin/blob/30a6c999351041d6a1e8712a9659be1296a1b46a/src/rpc/blockchain.cpp#L2344

In `generateblock()`:
https://github.com/bitcoin/bitcoin/blob/30a6c999351041d6a1e8712a9659be1296a1b46a/src/rpc/mining.cpp#L326

In `addconnection()`:
https://github.com/bitcoin/bitcoin/blob/30a6c999351041d6a1e8712a9659be1296a1b46a/src/rpc/net.cpp#L392